portal/resources/views/admin/participant/list.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('layouts.admin')
+
+@section('content')
+<div
+    class="container"
+    id="participant-search"
+    data-url={{ route('participant.show', ['participant' => '_participant_'])  }}
```

Comment:
Looks like parameter escaping is missing.

```
data-url="{{ route('participant.show', ['participant' => '_participant_']) }}"
```

Also not quite sure what you mean with `_participant_`

Reply:
The `_participant_` is replaced by the JS code (it's so the JS code can use a route). I think the quotation marks here are omitted for a good reason (probably because they were printed already) but I would have to check that to be sure.